tests/integration/pilot/main_test.go

```diff
@@ -35,7 +35,13 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
 		RequireSingleCluster().
-		Setup(istio.Setup(&i, nil)).
+		Setup(istio.Setup(&i, func(cfg *istio.Config) {
+			cfg.ControlPlaneValues = `
+values:
+  global:
+    meshExpansion:
+      enabled: true`
```

Comment:
Will this mean that all tests - even those without VMs - will have meshExpanshion enabled? IIRC this will actually change networking related settings which I don't think we recommend doing unless necessary.

Reply:
Thanks for the comment! Yes, this means that all tests (in pilot suite) will have meshExpansion enabled. However, since we will be extending a number of tests in this suite, having a separate installation of istio for these tests might not be entirely worth it. Also, the meshExpansion only opens up some points for the instances, so it is not that likely for it to interfere with other tests. For now, I think we could keep this in the main_test.go file, but I am happy to discuss this further with you, though. Please let me know if you still think this is necessary. Thanks!

Reply:
Lets follow up with adding VM related coverage as a feature. Can you open an issue and start the discussion?

Reply:
Whoops, added to the wrong comment. Please disregard.